siemens_data.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import sys
+import pandas as pd
+from datetime import datetime
+
+""" Class for reading and manipulating Siemens data from transformed CSVs. 
```

Comment:
Not a big deal for now, but docstrings for classes and functions usually go underneath the function, eg,

```python
class SiemensData:
    """ Class for reading and manipulating Siemens data from transformed CSVs.
    Contains API for accessing data, as well as function for reading the CSV's.
    """
    data = None
    datetimes = []
    ...

Reply:
Will fix this for the next commit!